src/parser/odoc_parser.mli

```diff
@@ -43,3 +43,6 @@ val position_of_point : t -> Loc.point -> Lexing.position
     the usual representation in the Lexing module. Note that this relies on the
     information passed in {!parse_comment}, and hence requires the result of
     that call in addition to the {!Loc.point} being converted. *)
+
+val codeblock_content : Loc.span -> string -> string * Warning.t list
+val verbatim_content : Loc.span -> string -> string * Warning.t list
```

Comment:
These will need some doc comments, plus we'll need to add something to the `Code_block in ast.mli